doc/conf_ref.rst

```diff
@@ -222,6 +222,19 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     RPM debug scriptlet output level. One of: ``critical``, ``emergency``,
     ``error``, ``warn``, ``info`` or ``debug``. Default is ``info``.
 
+.. _search_enabled_metadata-label:
+
+``search_enabled_metadata``
```

Comment:
Well that means that assumeyes in some cases will mean No. Funny staff around, isn't it.